sharktank/sharktank/layers/configs/llm_configs.py

```diff
@@ -121,7 +121,8 @@ class LlamaModelConfig:
 
     # Block sequence stride for a paged KV cache. This must divide evenly
     # into the context length.
-    block_seq_stride: int = 16
+    default_block_seq_stride: ClassVar[int] = 16
```

Comment:
Why do we need `ClassVar `here? Seeing the `default_block_seq_stride` default feels bad.